lib/infobox/node.js

```diff
@@ -81,6 +81,54 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
     return d.firstseen.fromNow(true)
   }
 
+  function wifiChannelAlias(ch) {
+    var chlist = {
+      "1"   :"2412 MHz",
+      "2"   :"2417 MHz",
+      "3"   :"2422 MHz",
+      "4"   :"2427 MHz",
+      "5"   :"2432 MHz",
+      "6"   :"2437 MHz",
+      "7"   :"2442 MHz",
+      "8"   :"2447 MHz",
+      "9"   :"2452 MHz",
+      "10"  :"2457 MHz",
+      "11"  :"2462 MHz",
+      "12"  :"2467 MHz",
+      "13"  :"2472 MHz",
+      "36"  :"5180 MHz (Indoors)",
+      "40"  :"5200 MHz (Indoors)",
+      "44"  :"5220 MHz (Indoors)",
+      "48"  :"5240 MHz (Indoors)",
+      "52"  :"5260 MHz (Indoors/DFS/TPC)",
+      "56"  :"5280 MHz (Indoors/DFS/TPC)",
+      "60"  :"5300 MHz (Indoors/DFS/TPC)",
+      "64"  :"5320 MHz (Indoors/DFS/TPC)",
+      "100" :"5500 MHz (DFS) !!",
+      "104" :"5520 MHz (DFS) !!",
+      "108" :"5540 MHz (DFS) !!",
+      "112" :"5560 MHz (DFS) !!",
+      "116" :"5580 MHz (DFS) !!",
+      "120" :"5600 MHz (DFS) !!",
+      "124" :"5620 MHz (DFS) !!",
+      "128" :"5640 MHz (DFS) !!",
+      "132" :"5660 MHz (DFS) !!",
+      "136" :"5680 MHz (DFS) !!",
+      "140" :"5700 MHz (DFS) !!"
+    }
+    if (!(ch in chlist))
+      return ""
+    else
+      return chlist[ch]
+  }
+
+  function showWifiChannel(ch) {
+      if (!ch)
```

Comment:
danke. eslinkt meckert (erst) jetzt wegen der leerzeichen zwischen key und value in der channelliste. Ich werde die auch noch anpassen (müssen).